src/libsystemd/sd-varlink/sd-varlink.c

```diff
@@ -2496,7 +2496,7 @@ _public_ int sd_varlink_replyb(sd_varlink *v, ...) {
         return sd_varlink_reply(v, parameters);
 }
 
-static int varlink_reset_fds(sd_varlink *v) {
+_public_ int sd_varlink_reset_fds(sd_varlink *v) {
```

Comment:
Not something with this PR, but "see above" in comment should be "below", i.e. sd_varlink_error